docset/winserver2022-ps/addsdeployment/Test-ADDSForestInstallation.md

```diff
@@ -195,11 +232,12 @@ The acceptable values for this parameter are:
 - Windows Server 2012:  5 or Win2012
 - Windows Server 2012 R2:  6 or Win2012R2
```

Comment:
I can't comment in this PR on the other lines (they're too far from the changes), but please update all enum lists in this article to render the integer and string values as code.

```suggestion
- Windows Server 2012:  `5` or `Win2012`
- Windows Server 2012 R2:  `6` or `Win2012R2`
```